app/views/studies/_initialize_metadata_form.html.erb

```diff
@@ -35,8 +39,9 @@
       &nbsp;
       <%= f.label :use_metadata_convention_false, 'data-analytics-name': 'metadata-convention-optout' do %>
         <% if convention_required %>
+
           <a href="#/" style="color:#999" id="convention-decline-label-<%= f.object.id.to_s %>"
-             data-content="Using the convention is now required.  If this presents a particular issue for your study, please contact us at <a href='mailto:scp-support@broadinstitute.zendesk.com'>scp-support@broadinstitute.zendesk.com</a> so we can help or make any updates to the convention."
+             data-content="Using the convention is now required.  If this presents a particular issue for your study, please use this <a href='<%= exemption_form %>'>contact form</a> so we can assist you with your metadata."
```

Comment:
Tighter copy for consideration, and a hook to more closely and easily analyze this important funnel:

```suggestion
             data-content="The convention is required.  If this is a problem for your study, please <a data-analytics-name='exemption-contact-us-link' href='<%= exemption_form %>'>contact us</a>."
```